lib/services/app-scaffolding-extensions-service.ts

```diff
@@ -30,6 +30,12 @@ export class AppScaffoldingExtensionsService extends dependencyExtensionsService
 			let afterPrepareAction = () => {
 				return (() => {
 					this.npmInstall("glob-watcher@0.0.8").wait(); // HACK: With this we are able to make paths shorter with 20 symbols.
+					
+					let generatorBaseDependencies = require(path.join(this.appScaffoldingPath, "node_modules", "screen-builder-base-generator", "package.json")).dependencies;
+					_.each(generatorBaseDependencies, (value, key) => {
```

Comment:
You can simplify this by using:
```TypeScript
_.each(generatorBaseDependencies, (value, key) => this.npmInstall(`${key}@${value}`).wait());
```
Also we can speed it up by using different futures for each install, for example
```TypeScript
Future.wait(_.map(generatorBaseDependencies, (value, key) => this.npmInstall(`${key}@${value}`));
```
> NOTE: I've not tested this Future.wait(...) code here